src/replicated.ts

```diff
@@ -0,0 +1,23 @@
+// These are very WIP
+
+export interface ReplicatedApp {
+  config: ConfigSection[];
+  components: Component[];
+}
+
+export interface ConfigSection {
+  items: ConfigOption[];
+}
+
+export interface Component {
+  cluster_host_count: MinMax;
+}
+
+export interface MinMax {
+  min: number;
+  max: number;
+}
+export interface ConfigOption {
+  name: string;
+  type: string;
```

Comment:
config options can be nested for some types (`select_one`, `select_many`, `select`)

https://github.com/replicatedcom/replicated-config-render/tree/master/src/components

```js
  - name: multi
    type: select_many
    items:
    - name: first
      title: First
    - name: second
      title: Second
```